jira/client.py

```diff
@@ -1667,6 +1667,51 @@ def project_role(self, project, id):
 
     # Resolutions
 
+    def add_user_to_role(self, project, role, username):
+        """
+        Add the user ``username`` to the ``role`` in ``project``
+
+        :param project: ID or key of the project to get the role from
+        :param role: ID of the role where the user will be added
+        :param username: The user to add to the role.
+        """
+        data = {}
+        data['user'] = [username]
+        if isinstance(role, Number):
+            role = "%s" % role
+        url = self._get_url('project/' + project + '/role/' + role)
+
+        r = self._session.post(
+            url, data=json.dumps(data))
+        if r.status_code == 200:
+            return True
+        else:
+            logging.warning(
+                'Failed to add user %s to role. Got code: %s' % (username, r.status_code))
+            return r.status_code
```

Comment:
This function returns `bool` or `int` as a result. I think that function result must be consistent - either all time returns `bool` or `int`.